### ISSUE TITLE
[BUG] Web upload not working when file is separated into multiple files

### ISSUE THREAD

**Issue by enm4r** [paperless-ngx/paperless-ngx#1862]
### Description

Hi, 

uploading documents via web frontend does not work for me, similiar to #1682 but different behaviour:

When uploading a regular file via web uploader, it will be imported without issues. When uploading a file with a barcode to separate for, it won't.
The regular unsplit file is handled, then split, but the resulting split files are not consumed again. Saw no errors popping up.

I have specified a barcode value via `PAPERLESS_CONSUMER_BARCODE_STRING`, by which the file should be separated, since the file contains multiple barcodes (each page a different one). The file is of type TIFF. As far as I looked into the files and understood the barcode processing, this should not cause any problems.

:information_source: **Note:** This behaviour does not occurr when using the consumption directory. Also does not occurr in version 1.8

### Steps to reproduce

1. Open web frontend
2. Upload a file via file upload in web frontend

### Webserver logs

```bash
[2022-10-25 10:43:30,983] [DEBUG] [paperless.barcodes] Detected mime type: image/tiff

[2022-10-25 10:43:30,984] [DEBUG] [paperless.barcodes] Detected mime type: image/tiff

[2022-10-25 10:43:31,573] [DEBUG] [paperless.barcodes] Barcode of type CODE128 found: 210481962

[2022-10-25 10:43:31,723] [DEBUG] [paperless.barcodes] Barcode of type CODE128 found: 210481942

[2022-10-25 10:43:31,871] [DEBUG] [paperless.barcodes] Barcode of type CODE128 found: 210481956

[2022-10-25 10:43:32,027] [DEBUG] [paperless.barcodes] Barcode of type CODE128 found: 210481921

[2022-10-25 10:43:32,171] [DEBUG] [paperless.barcodes] Barcode of type CODE128 found: 210481943

[2022-10-25 10:43:32,445] [DEBUG] [paperless.barcodes] Barcode of type CODE128 found: 210481939

[2022-10-25 10:43:32,590] [DEBUG] [paperless.barcodes] Barcode of type CODE128 found: 210481960

[2022-10-25 10:43:32,736] [DEBUG] [paperless.barcodes] Barcode of type CODE128 found: 210481930

[2022-10-25 10:43:32,889] [DEBUG] [paperless.barcodes] Barcode of type CODE128 found: 210481947

[2022-10-25 10:43:33,044] [DEBUG] [paperless.barcodes] Barcode of type CODE128 found: 210481932

[2022-10-25 10:43:33,045] [DEBUG] [paperless.tasks] Pages with separators found in: /tmp/paperless/paperless-upload-ha_39s36

[2022-10-25 10:43:33,053] [DEBUG] [paperless.barcodes] Starting new document at idx 1

[2022-10-25 10:43:33,053] [DEBUG] [paperless.barcodes] Split into 2 new documents

[2022-10-25 10:43:33,055] [DEBUG] [paperless.barcodes] pdf no:0 has 1 pages

[2022-10-25 10:43:33,064] [DEBUG] [paperless.barcodes] pdf no:1 has 9 pages

[2022-10-25 10:43:33,071] [DEBUG] [paperless.barcodes] moving /tmp/paperless/paperless-upload-ha_39s36_document_0.pdf to /tmp/paperless/0_doc20220808104440.tif

[2022-10-25 10:43:33,074] [DEBUG] [paperless.barcodes] moving /tmp/paperless/paperless-upload-ha_39s36_document_1.pdf to /tmp/paperless/1_doc20220808104440.tif

[2022-10-25 10:43:33,075] [DEBUG] [paperless.tasks] Deleting file /tmp/paperless/paperless-upload-ha_39s36
```

### Paperless-ngx version

1.9.2

### Host OS

Debian

### Installation method

Docker - official image

### Browser

Firefox

### Configuration changes

USERMAP_UID=1001 USERMAP_GID=1001 PAPERLESS_TIME_ZONE=Europe/Berlin PAPERLESS_OCR_LANGUAGE=deu+eng PAPERLESS_SECRET_KEY=KtmPsr6CeIzCFSBFwqnJ51AiqFVirLzsCy30XzQVzHXqIcXDlCb82z98bMYWpwVh PAPERLESS_TASK_WORKERS=8 PAPERLESS_FILENAME_FORMAT={created_year}/{title} PAPERLESS_POST_CONSUME_SCRIPT=/usr/src/paperless/scripts/post_consumption.py PAPERLESS_CONSUMER_ENABLE_BARCODES=true PAPERLESS_CONSUMER_BARCODE_TIFF_SUPPORT=true PAPERLESS_CONSUMER_BARCODE_STRING=210481942

**Comment by gador** [comment]
This should be fixed by #1762

**Comment by enm4r** [comment]
Thanks @gador. Will wait for the next update then 👍
